ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -703,9 +712,17 @@ export class TunnelManagementHttpClient implements TunnelManagementClient {
         this.raiseReportProgress(TunnelProgress.StartingSendTunnelRequest);
         const uri = await this.buildUri(clusterId, path, query, options);
         const config = await this.getAxiosRequestConfig(undefined, options);
-        const result = await this.request<TResult>(method, uri, body, config, allowNotFound);
-        this.raiseReportProgress(TunnelProgress.CompletedSendTunnelRequest);
-        return result;
+        try {
+            const result = await this.request<TResult>(method, uri, body, config, allowNotFound);
+            this.raiseReportProgress(TunnelProgress.CompletedSendTunnelRequest);
+            return result;
+        } catch (error) {
+            if (/certificate/i.test((error as Error).message)) {
+                throw new Error("Tunnel service HTTPS certificate is invalid. This may be caused by the use of a "+
+                "self signed certificate or a firewall intercepting the connection.");
```

Comment:
Is there a need to include the original error message here also?

Reply:
When I run this locally my exception message is just "self signed certificate". I think that would not be descriptive enough to add as an addition to this exception message, but if it would mean consistency between the ts and cs files then I can add it